server/controllers/reports_proposed/data/debitor_group_report.js

```diff
@@ -0,0 +1,75 @@
+// reports_proposed/data/balance_sheet.js
+// Collects and aggregates data for the enterprise balance sheet
+var q       = require('q');
+var db      = require('../../../lib/db');
+var numeral = require('numeral');
+
+var formatDollar = '$0,0.00';
+var debitorGroupReportDate = new Date();
+
+exports.compile = function (options) {
+  'use strict';
+
+  var i18nDebitorGroupReport = options.language == 'fr' ? require('../lang/fr.json').DEBITOR_GROUP_REPORT : require('../lang/en.json').DEBITOR_GROUP_REPORT;
+  var deferred = q.defer(), context = {}, params = options.involveJournal === true ? [options.dg.account_id, options.dg.account_id] : [options.dg.account_id];
+
+  var sql2 =
```

Comment:
Could you use a more meaningful name than sql2? Also, why does sql2 come before sql in the definitions?